docs/queries/examples.md

```diff
@@ -46,6 +56,15 @@ All done tasks that are anywhere in the vault under a `tasks` heading (e.g. `##
 
 ---
 
+All tasks done this month but that were due or scheduled the month before:
+
+    ```tasks
+    (due before this month) OR (scheduled before this month)
+    done this month
+    ```
+
+---
+
```

Comment:
Another nice example.

> due or scheduled the month before

I am not sure that it is really 'the month before' as I think that ones due or scheduled in earlier months will also match.

Perhaps something like:

`All tasks done this month but that were due or scheduled in earlier months:`

Reply:
I realised that the query and the description are not matching so I modified the both and took into account your previous comment, lmk if this is better

Reply:
It's great - thank you.